docs/kiwix-serve.rst

```diff
@@ -51,7 +51,11 @@ Options
 .. option:: -i ADDR, --address=ADDR
 
   Listen only on this IP address. By default the server listens on all
-  available IP addresses.
+  available IP addresses. Alternatively, you can use special values to define which types of connections to accept:
+
+  all : Listen for connections from all IP addresses (IPv4 and IPv6).
+  ipv4 : Listen for connections from all IPv4 addresses.
+  ipv6 : Listen for connections from all IPv6 addresses.
```

Comment:
1. from → on
2. Make it a bulleted list (start each line with a `- `). Build (via `ninja doc`) and check the documentation to make sure that it looks nice.